src/redux/constants.js

```diff
@@ -0,0 +1,62 @@
+const SET_POLYGON_DATA = 'SET_POLYGON_DATA';
+const SET_BOTTOM_DRAWER_DATA = 'SET_BOTTOM_DRAWER_DATA';
+
+const TOGGLE_LEFT_DRAWER = 'TOGGLE_LEFT_DRAWER';
+const TOGGLE_RIGHT_DRAWER = 'TOGGLE_RIGHT_DRAWER';
+const TOGGLE_BOTTOM_DRAWER = 'TOGGLE_BOTTOM_DRAWER';
+
+const LEFT_PANEL_VENDOR_SELECT = 'LEFT_PANEL_VENDOR_SELECT';
+const RIGHT_PANEL_VENDOR_SELECT = 'RIGHT_PANEL_VENDOR_SELECT';
+const MAP_VENDOR_SELECT = 'MAP_VENDOR_SELECT'
+
+const FIND_MY_LOCATION_SELECT = 'FIND_MY_LOCATION_SELECT';
+const FIND_MY_LOCATION_ERROR = 'FIND_MY_LOCATION_ERROR';
+const FIND_MY_LOCATION_SUCCESS = 'FIND_MY_LOCATION_SUCCESS';
+const FIND_MY_LOCATION_OUT_OF_BOUNDS = 'FIND_MY_LOCATION_OUT_OF_BOUNDS';
+
+const SET_MAP = 'SET_MAP';
+
+const SET_TAB_VALUE = 'SET_TAB_VALUE';
+
+// TODO put this in a better place
+const drawerWidth = 280;
+
+export {
+  SET_POLYGON_DATA,
+  SET_BOTTOM_DRAWER_DATA,
+  TOGGLE_LEFT_DRAWER,
+  TOGGLE_RIGHT_DRAWER,
+  TOGGLE_BOTTOM_DRAWER,
+  LEFT_PANEL_VENDOR_SELECT,
+  RIGHT_PANEL_VENDOR_SELECT,
+  MAP_VENDOR_SELECT,
+  FIND_MY_LOCATION_SELECT,
+  FIND_MY_LOCATION_ERROR,
+  FIND_MY_LOCATION_SUCCESS,
+  FIND_MY_LOCATION_OUT_OF_BOUNDS,
+  SET_TAB_VALUE,
+  SET_MAP,
+  drawerWidth
+}
+
+// TODO put this in a better place
+export const breweries = [
```

Comment:
Is this seafood fest specific? If so we should probably remove to avoid confusion.

Reply:
Yeah for sure